src/libs/ReportUtils.js

```diff
@@ -90,6 +90,22 @@ function getChatType(report) {
     return report ? report.chatType : '';
 }
 
+/**
+ * Returns the report action with key parentReportID.
+ * This function is to be replaced by a HOC.
+ *
+ * @param {Object} parentReportActions
+ * @param {string} parentReportID
+ * @returns {Object}
+ */
+function getParentReportAction_DEV(parentReportActions, parentReportID) {
+    if (!parentReportActions) {
+        return {};
+    }
+    const matchingKey = _.find(_.keys(parentReportActions), (key) => _.isEqual(key, parentReportID));
+    return matchingKey ? parentReportActions[matchingKey] : null;
+}
```

Comment:
```suggestion
function getParentReportAction_DEV(parentReportActions, parentReportActionID) {
    return lodashGet(parentReportActions, [parentReportActionID], {});
}
```

Reply:
Looking at this again I think we can merge `getReportActions` and `getParentReportAction_DEV` into one function. But since the later function may be integrated into a HOC I'm not sure about that.